app/core/layouts/Header.tsx

```diff
@@ -9,6 +9,7 @@ import { useOptionalUser } from "~/utils";
 interface IProps {
   title: string;
   existApplicant?: boolean;
+  applicantId?: string | number;
```

Comment:
Maybe we can just have applicantId, if we allow `undefined` we can check if its set or not, instead of using `existApplicant`